client/i18next-parser.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
This file is used once while keys are generated first time. It's not necessary to keep it in repository.